pages/common/git-grep.md

```diff
@@ -0,0 +1,21 @@
+# git-grep
+
+> Find strings inside files anywhere in a repository's history.
+> Accepts a lot of the same flags as regular `grep`.
+> Documentation: <https://git-scm.com/docs/git-grep>.
+
+- Search for a string in tracked files:
+
+`git grep {{search_string}}`
+
+- Search for a string in files matching a pattern in tracked files:
+
+`git grep {{search_string}} -- {{file_glob_pattern}}`
+
+- Search for a string in tracked files, including submodules:
+
+`git grep --recurse-submodules {{search_string}}`
+
+- Search for a string in at a specific point in history:
```

Comment:
Perhaps this will read better?
```suggestion
- Search for a string at a specific point in history:
```

Reply:
@sbrl, should this just be committed as it's such a minor change?